libs/community/langchain_community/vectorstores/aerospike.py

```diff
@@ -545,7 +545,7 @@ def from_texts(
         cls,
         texts: List[str],
         embedding: Embeddings,
-        metadatas: Optional[List[dict]] = None,
+        labels: Optional[List[dict]] = None,
```

Comment:
breaking change

Reply:
Our client was still in beta when this was initially created. This breaking change is OK since it better matches our documentation. If you still would rather we not commit this breaking change, I will revert it.